packages/x-date-pickers/src/internals/hooks/usePicker/usePicker.types.ts

```diff
@@ -115,6 +115,7 @@ export interface UsePickerNonStaticProps extends Omit<PickerFieldPrivateContextV
   format?: string;
   /**
    * If `true`, the button to open the Picker will not be rendered (it will only render the field).
+   * @deprecated Use the field components instead (for example, <DateField /> instead of <DatePicker disableOpenPicker />).
```

Comment:
It feels slightly strange mentioning DateField on every different Picker component doc and JSDoc. 🤔 
WDYT about going for a link to the Fields concept doc? 🤔 
```suggestion
   * @deprecated Use the [field component](https://next.mui.com/x/react-date-pickers/fields/) instead.
```